primitives/session/src/lib.rs

```diff
@@ -118,10 +118,12 @@ where
 	T: ProvideRuntimeApi<Block>,
 	T::Api: SessionKeys<Block>,
 {
-	let runtime_api = client.runtime_api();
+	if !seeds.is_empty() {
```

Comment:
Isn't it easier to:
```
if seeds.is_empty() {
  return Ok(())
}
```

Reply:
A question of style, after compilation it is strictly equivalent.

Reply:
Yeah. But stylistically maybe is better to early exit when the initial conditions are not met (just to avoid a dummy indentation over all the function body)

But this is only my opinion :-D

Reply:
Ok